mobile_back_tflite/cpp/backend_tflite/tflite_c.cc

```diff
@@ -260,7 +260,7 @@ mlperf_backend_ptr_t mlperf_backend_create(
     } else if (strcmp(configs->accelerator, "neuron") == 0) {
       // The kOptimizationBatchProcessor doesn't work yet.
       // Use NNAPI instead.
-      if (configs->batch_size != 1) {
+      if (configs->batch_size > 1) {
```

Comment:
Shouldn't we use `backend_data->real_batch_size` here?

Reply:
Nope, `backend_data->real_batch_size` also works for current hardware, but it's not what we want.

If I I have 4 NPUs which perform best with batch size one (4 shards/threads, real_batch_size == 1), I still want to use NNAPI delegate rather than Neuron delegate. 

By the way, it seems the batch_size (128 for neuron backend) in backend setting file is overwritten by numbers from UI (4 threads, batch size: 2).

Reply:
>Nope, backend_data->real_batch_size also works for current hardware, but it's not what we want.

Could you add a comment about this in the code, to make sure no one tries to "fix" it?

>By the way, it seems the batch_size (128 for neuron backend) in backend setting file is overwritten by numbers from UI (4 threads, batch size: 2).

Yep. If I remember correctly, Flutter app didn't use batch size at all at first, and when we fixed it, we were optimizing it for iPhones.
We should probably rework this behavior at some point, maybe in January.